package.json

```diff
@@ -17,21 +17,10 @@
     "@material-ui/lab": "*",
     "@plasmicapp/plume": "^0.1.20",
     "@plasmicapp/react-web": "^0.2.61",
-    "@testing-library/jest-dom": "^5.14.1",
```

Comment:
No big changes here, just small updates and moving dependencies which would only be used in development to `devDependencies`.